src/api/Client/WinMM/winmmdrv.cpp

```diff
@@ -52,8 +52,10 @@ struct MIDICLIENT
     LPMIDIHDR buffers[16] {0};
 };
 
-struct MIDICLIENT g_InClients[32];
-struct MIDICLIENT g_OutClients[32];
+#define MAX_CLIENTS 32

```

Comment:
Is 32 a current limitation as well? Seems quite low if there's a client per port.